File: Android Kotlin Retrofit Node Express- Orphanslife/app/src/main/java/com/kaligotla/orphanslife/model/entity/Child.kt
package com.kaligotla.orphanslife.model.entity

import androidx.room.Embedded
import androidx.room.Entity
import androidx.room.PrimaryKey
import com.google.gson.annotations.SerializedName

@Entity(
    tableName = "child"
)

data class Child(

    @PrimaryKey
    @SerializedName("child_id")
    val child_id: Int,

    @SerializedName("child_name")
    val child_name: String,

    @SerializedName("child_dob")
    val child_dob: String,

    @SerializedName("child_gender")
    val child_gender: String,

    @SerializedName("admitted_date")
    val admitted_date: String,

    @SerializedName("leave_date")
    val leave_date: String,

    @SerializedName("mother_name")
    val mother_name: String,

    @SerializedName("father_name")
    val father_name: String,

    @SerializedName("child_mobile")
    val child_mobile: String,

    @SerializedName("child_image")
    val child_image: String,

    @SerializedName("status_id")
    val status_id: Int,

    @SerializedName("admin_id")
    val admin_id: Int,

    @SerializedName("child_created_at")
    val child_created_at: String,

    @SerializedName("child_updated_at")
    val child_updated_at: String
)

File: MongoDB Express - API - Orphanslife/Data/model/Child.js
const mongoose = require('mongoose')
const Schema = mongoose.Schema
const passportLocalMongoose = require('passport-local-mongoose');

var Child = new Schema({
    child_id:{
        type: Number
    },
    child_name:{
        type: String
    },
    child_dob:{
        type: String
    },
    child_gender:{
        type: String
    },
    admitted_date:{
        type: String
    },
    leave_date:{
        type: String
    },
    mother_name:{
        type: String
    },
    father_name:{
        type: String
    },
    child_mobile:{
        type: String
    },
    child_image:{
        type: String
    },
    status_id:{
        type: Number
    },
    admin_id:{
        type: Number
    },
    child_created_at:{
        type: String
    },
    child_updated_at:{
        type: String
    }
}, { collection: 'child' })
  
Child.plugin(passportLocalMongoose);
  
module.exports = mongoose.model('Child', Child)